### ISSUE TITLE
Treat Notes as manual loot

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#378]
Nexus request - for research

> is there a way for the mod to display a message when near notes in general instead of auto picking/looting them? (like it does for Stones of Barenziah by default)

This is likely going to be difficult to automate because what is a Note is subjective

**Comment by Lywzc** [comment]
There is a mod for SkyUI that detects Note based on inventory image due to SE not using Note as a type.

**Comment by Lywzc** [comment]
Yeah. Although I am not interested in this request if you want to implement it you can try a similar way.

**Comment by SteveTownsend** [comment]
Thanks for the pointer. Did you see that the recent Ghost changes broke
some NPCs, by the way?

**Comment by Lywzc** [comment]
Yes, I looked into that and will reply there.

**Comment by SteveTownsend** [comment]
Thanks. Probably just need to tune the updated logic a little to handle
both cases properly
